docs/install.md

```diff
@@ -113,3 +113,49 @@ spanner each time follow the following steps:
     ```sh
     export SPANNER_EMULATOR_HOST=localhost:9010
     ```
+
+## Quickstart
+
+[GCP Cloud shell](https://cloud.google.com/shell) is an online development and operations environment accessible anywhere with your browser. You can manage your resources with its online terminal preloaded with utilities such as the gcloud command-line tool, kubectl, and more. It is free to run, and can help you get started with Spanner migration tool quickly. Follow the instructions below to get started:
+
+1. Login to Google Cloud console and click on the Cloud shell button on the top right corner.
+
+    ![cloud shell](https://services.google.com/fh/files/misc/cloud_shell.png)
+
+2. This is launch a terminal on Cloud console. Wait for the terminal to load.
+
+    ![provision](https://services.google.com/fh/files/misc/provision.png)
+
+3. Once the terminal is loaded, run the following command:
+
+    ```sh
+    sudo apt-get install google-cloud-sdk-spanner-migration-tool
+    ```
+
+4. Wait for the installation to complete. Spanner migration tool can be accessed using the following gCloud CLI command:
+
+    ```sh
+    gcloud alpha spanner migrate <COMMAND>
+    ```
+
+5. In order to launch the web UI for Spanner migration tool, run the following command:
+
+    ```sh
+    gcloud alpha spanner migrate web
+    ```
+
+    {: .important }
+    You will be asked to authorize running this command by providing your Google Cloud credentials. This allows Spanner migration tool to access resources on your behalf.
+
+6. This is launch the web UI in the Cloud shell instance, and show the following message in the terminal logs:
+
+    ```sh
+    Starting Spanner migration tool UI at: http://localhost:8080
+    ```
+
+7. **Without exiting the Cloud shell command**, click on the "Web preview" icon on the top right corner of the cloud shell terminal, and then click on "Preview on port 8080". This will open the Spanner migration tool UI in a new tab. The web address of the UI would something like `https://8080-cs-<random-string>.cs-<region>-vwey.cloudshell.dev/`.
```

Comment:
This should probably be moved to another page around how to use the Cloud Shell. We should also include how to connect to your local DB using this - including IP allowlisting etc.

We can link that page here.

Reply:
Will include the local DB bits. 

Do you have a suggestion on where to move this? `Installation` page seemed like the most obvious choice here...